index.html

```diff
@@ -36,237 +36,257 @@ <h4>My Logo</h4>
+            <img src="images/Linkedin.svg" alt="Lilian's LinkedIn" />
+          </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://medium.com/@chatwitlily/">
+            <img src="images/medium.svg" alt="Lilian's Medium" /> </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://angel.co/u/lilian-francis"> <img src="images/angellist.svg" alt="Lilian's AngelList" />
+          </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://github.com/ifnotlily/">
+            <img src="images/github.svg" alt="Lilian's Github" />
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div id="iteme-7"><button class="buttonresume">Get my Resume</button></div>
+    <div class="my-accordion">
+      <ul>
+        <li class='menu'>Languages
 
 
-          <li class='submenu'><a href=""><img src="/images/Ellipse1.png" alt="javascript">JavaSript</a></li>
-          <li class='submenu'><a href=""><img src="/images/Ellipse2.png" alt="html">HTML</a></li>
-          <li class='submenu'><a href=""><img src="/images/Ellipse3.png" alt="css">CSS</a>
-          </li>
+        <li class='submenu'><a href=""><img src="/images/Ellipse1.png" alt="javascript">JavaSript</a></li>
+        <li class='submenu'><a href=""><img src="/images/Ellipse2.png" alt="html">HTML</a></li>
+        <li class='submenu'><a href=""><img src="/images/Ellipse3.png" alt="css">CSS</a>
+        </li>
 
 
-          <li class='menu'>Frameworks<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
+        <li class='menu'>Frameworks<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
 
 
-          <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
-        </ul>
+        <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
+      </ul>
 
+    </div>
 
-      </div>
+  </section>
+  <section class="contactform">
+    <div class="container"></div>
+    <div class="content-form">
+      <h3>Contact me</h3>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="#"
+        method="post">
+        <fieldset id="fs-frm-inputs">
+          <input type="text" name="name" id="full-name" placeholder="Full name" required="">
+          <input type="email" name="_replyto" id="email-address" placeholder="Email address" required="">
+          <textarea rows="5" name="message" id="message" placeholder="Enter text here" required=""></textarea>
+          <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
+        </fieldset>
+        <input type="submit" value="Get in touch" class="submit-button">
```

Comment:
- [ ] Remember the button should follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) when this is pressed, hover, etc. Please work on it. 🐦 

![image](https://user-images.githubusercontent.com/20563252/126349446-d2a15777-7387-42cd-8c08-8a4dd28d2a0d.png)